src/tokenizer.ts

```diff
@@ -11,3 +11,30 @@ export function getTokenCount(input: string): number {
   input = input.replace(/<\|endoftext\|>/g, '')
   return encode(input).length
 }
+
+export function splitPrompt(
```

Comment:
The splitprompt function seems to be splitting the file diff randomly based on tokencount  / limit. But that means when the diff is sent to openai - it won't know the remaining diff or previous diff. This may result in incorrect review comments being posted. 
Or is this getting handled somehow?